idioms/mem-take.md

```diff
@@ -95,6 +99,10 @@ borrow checker. The compiler may fail to optimize away the double store,
 resulting in reduced performance as opposed to what you'd do in unsafe
 languages.
 
+The type you are taking needs to implement the [`Default` trait](./default.md).
```

Comment:
```suggestion
Furthermore the type you are taking needs to implement the [`Default` trait](./default.md).
```

Reply:
Committed in 0eaa3ba, with an added comma.